mkdocs.yml

```diff
@@ -93,6 +93,41 @@ plugins:
         'unified/novacustom/rgb-keyboard.md': 'unified/novacustom/features.md'
+        'variants/protectli_fw6/overview.md': 'unified/protectli/overview.md#fw6'
+        'variants/protectli_vp1x1x/overview.md': 'unified/protectli/overview.md#v1000-series'
+        'variants/protectli_vp46xx/overview.md': 'unified/protectli/overview.md#vp46xx'
+        'variants/protectli_vp66xx/overview.md': 'unified/protectli/overview.md#vp66xx'
+        'variants/protectli_vp2410/overview.md': 'unified/protectli/overview.md#vp2410'
+        'variants/protectli_vp2420/overview.md': 'unified/protectli/overview.md#vp2420'
+
+        'variants/protectli_fw6/releases.md': 'unified/protectli/releases.md#fw6'
+        'variants/protectli_vp1x1x/releases.md': 'unified/protectli/releases.md#v1000-series'
+        'variants/protectli_vp46xx/releases.md': 'unified/protectli/releases.md#vp46xx'
+        'variants/protectli_vp66xx/releases.md': 'unified/protectli/releases.md#vp66xx'
+        'variants/protectli_vp2410/releases.md': 'unified/protectli/releases.md#vp2410'
+        'variants/protectli_vp2420/releases.md': 'unified/protectli/releases.md#vp2420'
+
+        'variants/protectli_fw6/building_manual.md': 'unified/protectli/building_manual.md#fw6'
+        'variants/protectli_vp1x1x/building_manual.md': 'unified/protectli/building_manual.md#v1000-series'
+        'variants/protectli_vp46xx/building_manual.md': 'unified/protectli/building_manual.md#vp46xx'
+        'variants/protectli_vp66xx/building_manual.md': 'unified/protectli/building_manual.md#vp66xx'
+        'variants/protectli_vp2410/building_manual.md': 'unified/protectli/building_manual.md#vp2410'
+        'variants/protectli_vp2420/building_manual.md': 'unified/protectli/building_manual.md#vp2420'
+
+        'variants/protectli_fw6/firmware_update.md': 'unified/protectli/firmware_update.md#fw6'
+        'variants/protectli_vp1x1x/firmware_update.md': 'unified/protectli/firmware_update.md#v1000-series'
+        'variants/protectli_vp46xx/firmware_update.md': 'unified/protectli/firmware_update.md#vp46xx'
+        'variants/protectli_vp66xx/firmware_update.md': 'unified/protectli/firmware_update.md#vp66xx'
+        'variants/protectli_vp2410/firmware_update.md': 'unified/protectli/firmware_update.md#vp2410'
+        'variants/protectli_vp2420/firmware_update.md': 'unified/protectli/firmware_update.md#vp2420'
+
+        'variants/protectli_fw6/recovery.md': 'unified/protectli/recovery.md#fw6'
+        'variants/protectli_vp1x1x/recovery.md': 'unified/protectli/recovery.md#v1000-series'
+        'variants/protectli_vp46xx/recovery.md': 'unified/protectli/recovery.md#vp46xx'
+        'variants/protectli_vp66xx/recovery.md': 'unified/protectli/recovery.md#vp66xx'
+        'variants/protectli_vp2410/recovery.md': 'unified/protectli/recovery.md#vp2410'
+        'variants/protectli_vp2420/recovery.md': 'unified/protectli/recovery.md#vp2420'
```

Comment:
initial deployment missing here

some of the post-installation setup as well, but not sure if it was ever linked to before

Reply:
Added initial deployment and post-installation for the platforms which are concerned.
Also sorted the redirects alphabetically by the `*.md` file basename.
https://github.com/Dasharo/docs/compare/9702d18e14289c37406c415ba4b848709fd768cf..7800087bfc37faf292cdb6ef6750eeba07f163de#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2R97-R142